website/docs/r/compute_health_check.html.markdown

```diff
@@ -10,7 +10,7 @@ description: |-
 
 Manages a health check within GCE. This is used to monitor instances
 behind load balancers. Timeouts or HTTP errors cause the instance to be
-removed from the pool. For more information, see [the official
+removed from the pool. This cannot be used with GCE `TargetPools` as they only support `HttpHealthChecks`. For more information, see [the official
```

Comment:
I agree that this behavior is confusing.

However it would be nicer to make the documentation match the public API closer. Maybe something like:

```markdown
Be aware that if you're managing a [Network Load Balancer](https://cloud.google.com/compute/docs/load-balancing/network/), you must use one of `google_compute_http_health_check` or `google_compute_https_health_check` instead of this resource. See [Legacy Health Checks](https://cloud.google.com/compute/docs/load-balancing/health-checks#legacy_health_checks) for more information.
```